src/RoboChat.Library/RoboChat.cs

```diff
@@ -101,5 +101,7 @@ private string GetRandomMessage(IEnumerable<ValidSentence> validSentences)
         {
             return validSentences.Skip(new Random().Next(validSentences.Count())).First().Message;
         }
+
+        public int NumberOfMessagesInCurrentSession => currentSessionMessagesHistory.Count;
```

Comment:
To powinno być na górze pliku, spójrz jak jest tworzona pełna historia (bazowa + z sesji)
w tamtym miejscu to daj

Reply:
Powinno być tylko w celu przejrzystości?